cryodrgn/fft.py

```diff
@@ -3,7 +3,7 @@
 try:
     import cupy as cp
 except ImportError:
-    cp = None
+    cp = np
 
 
 def fft2_center(img):
```

Comment:
This line `pp = np if ininstance(img, np.ndarray) else cp` is pretty ugly. What type should it be for `cp`? Would be better to say `pp = get_fftlib(img)` and have `get_fftlib` specifically check for the right types for each library and raise if it doesn't match.